src/user.h

```diff
@@ -189,6 +189,7 @@ typedef struct fakeuser {
     char uid[UIDLEN + 1];
 } Fake;
 
+User *find_account(Nick *);
```

Comment:
Please constify to reflect the constify in the function definition.

Reply:
Done